tests/test_precompiled.py

```diff
@@ -200,7 +200,7 @@ def test_precompiled_validation_with_preview_throws_error_if_file_is_not_a_pdf(c
 
     assert response.status_code == 400
     json_data = json.loads(response.get_data())
-    assert json_data['message'] == 'unable-to-read-the-file'
+    assert json_data['message'] == 'Unable to read the PDF data: Could not read malformed PDF file'
```

Comment:
I couldn't quite work out why this has changed. I'm vaguely on the right lines but couldn't identify the exact line of code that changes the result. If you could point it out to me in person tomorrow that would be great, thanks!